homeassistant/components/tplink_eap/device_tracker.py

```diff
@@ -0,0 +1,135 @@
+"""Tracker for Wifi clients connected to a TP-Link EAP."""
```

Comment:
If you are not going to provide tests for this file, it should be put into the `.coveragerc` file.